server/src/test/java/io/crate/replication/logical/action/PublicationsStateActionTest.java

```diff
@@ -92,6 +98,7 @@ public boolean hasPrivilege(Privilege.Type type, Privilege.Clazz clazz, String i
         Map<RelationName, RelationMetadata> resolvedRelations = publication.resolveCurrentRelations(clusterService.state(), user, user, "dummy");
         assertThat(resolvedRelations.keySet(), contains(new RelationName("doc", "t1")));
         appender.assertAllExpectationsMatched();
+        clusterService.close();
```

Comment:
This `close` would not be called in case of a test failure and thus leading to thread leaks. Can we assign the new cluster service to the test instance var instead in order to safely let it close on tearDown?